packages/cf-deploy-config-writer/src/mta-config/mta.ts

```diff
@@ -515,14 +516,17 @@ export class MtaConfig {
      * Append and/or cleanup the destination resource if missing in mta.yaml.
      *
      * @param {boolean} isManagedApp - if true, append managed approuter configuration
+     * @param {boolean} doCleanUp - if true, will ensure any missing modules | resources are appended
```

Comment:
`doCleanUp` is a little ambiguous ? What about `addMissingModules`?

Reply:
Done